modules/perception/camera/app/cipv_camera.cc

```diff
@@ -584,17 +584,17 @@ bool Cipv::IsPointLeftOfLine(const Point2Df &point,
             << "), cross_product: " << cross_product;
     }
     return true;
-  } else {
-    if (debug_level_ >= 2) {
-      AINFO << "point (" << point(0) << ", " << point(1)
-            << ") is right of line_segment (" << line_seg_start_point(0) << ", "
-            << line_seg_start_point(1) << ")->(" << line_seg_end_point(0)
-            << ", " << line_seg_end_point(1)
-            << "), cross_product: " << cross_product;
-    }
-    return false;
   }
+  if (debug_level_ >= 2) {
+    AINFO << "point (" << point(0) << ", " << point(1)
```

Comment:
AINFO_IF

Reply:
How about post another PR to make other such kinds of stuff to use AINFO_IF since the current PR is intended to refine a couple of logics.